Main.py

```diff
@@ -171,21 +177,27 @@ def login():
     else:
         email = request.form['email']
         password = request.form['password']
-        data = User.query.filter_by(email=email,
-                                    password=password).first()
+        #data = User.query.filter_by(email=email,password=password).first()
+        #query based on email and to check whether the password entered by the user is correct or not
+        data = User.query.filter_by(email=email).first()
```

Comment:
Hey here we need to check for both the email and password of the user (**password** is missing)